SalesforceSDK/Salesforce.SDK.TypeScript/data/bootconfig.json

```diff
@@ -0,0 +1,10 @@
+﻿{
```

Comment:
I'm okay with this for now. We need to ensure ```bootconfig.json``` stays in sync with the other hybrid apps (when stuff changes in the ```shared``` repo).

@kchitalia At some point, we need to explore the submodule approach.